src/v/pandaproxy/client/test/fetch.cc

```diff
@@ -94,7 +94,6 @@ FIXTURE_TEST(pandaproxy_fetch, ppc_test_fixture) {
         auto const& r = p.responses[0];
         BOOST_REQUIRE_EQUAL(r.id, ntp.tp.partition);
         BOOST_REQUIRE_EQUAL(r.error, kafka::error_code::none);
-        BOOST_REQUIRE_EQUAL(r.high_watermark, committed_offset);
```

Comment:
seems ok, but also seems like a workaround for something deeper. the commit doesn't say why is ok to *not* check the high watermark. w/ this equality. is that a bug in the test? can you add a bit of explanation.

Reply:
The endpoint doesn't do anything with this number, it was being over-tested. I can alter the commit message.

Reply:
sounds good if you resubmit the commit message, i'll approve.